test/features/jarvis-chat/jarvis-chat.test.js

```diff
@@ -70,11 +70,92 @@ describe('Jarvis Chat', () => {
     expect(args.region).to.equal('africa');
   });
 
+  it('should receive the correct custom surface id configuration', async () => {
+    setConfig(defaultConfig);
+    const config = Object.assign(getConfig(), {
+      locale: {
+        ietf: 'en',
+        prefix: '/africa',
+      },
+    });
+
+    const testSurfaceId = 'test-id';
+    const getMetadataMock = sinon.stub();
+    getMetadataMock.withArgs('jarvis-surface-id').returns(testSurfaceId);
+    await initJarvisChat(config, sinon.stub(), sinon.stub(), getMetadataMock);
+    const args = initializeSpy.getCall(0).args[0];
+    expect(args.appid).to.equal(testSurfaceId);
+    expect(args.appver).to.equal(defaultConfig.jarvis.version);
+    expect(args.env).to.equal(config.env.name === 'prod' ? 'prod' : 'stage');
+  });
+
+  it('should receive the correct custom version configuration', async () => {
+    setConfig(defaultConfig);
+    const config = Object.assign(getConfig(), {
+      locale: {
+        ietf: 'en',
+        prefix: '/africa',
+      },
+    });
+    const testVersion = '0.123';
+    const getMetadataMock = sinon.stub();
+    getMetadataMock.withArgs('jarvis-surface-version').returns(testVersion);
+    await initJarvisChat(config, sinon.stub(), sinon.stub(), getMetadataMock);
+    const args = initializeSpy.getCall(0).args[0];
+    expect(args.appid).to.equal(defaultConfig.jarvis.id);
+    expect(args.appver).to.equal(testVersion);
+    console.log('args.appid: ', args.appid);
+    console.log('args.appver: ', args.appver);
```

Comment:
We probably shouldn't pollute the unit test output with additional messages that don't have a clear goal

Reply:
removed the console logs